packages/cli-platform-apple/src/tools/__tests__/listDevices.test.ts

```diff
@@ -7,7 +7,8 @@
  */
 
 import execa from 'execa';
-import listIOSDevices from '../listIOSDevices';
+import listDevices from '../listDevices';
+import {getPlatformInfo} from '../../commands/runCommand/getPlatformInfo';
```

Comment:
Looks like this method is here by accident. Can we create a dedicated `commands/runCommand/__tests__/getPlatformInfo.test.ts`?